src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+import { findUser } from '../../utils/functions';
+import users from '../../api/users';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = findUser(todo.userId, users);
+
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed === true,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={user} />
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
add this check here, and don't render the component if the user doesn't exist at all